release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -1351,6 +1351,8 @@ $ ./openshift-install wait-for install-complete
 
 * Due to an unresolved metadata API issue, you cannot install clusters that use bare-metal workers on {rh-openstack} 16.1. Clusters on {rh-openstack} 16.2 are not impacted by this issue. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2033953[*BZ#2033953*])
 
+* Clusters that run on {rh-openstack} and use the OVN load-balancer provider driver do not support load balancers that include the `loadBalancerSourceRanges` attribute. There is no workaround for this issue. (link:https://issues.redhat.com/browse/OCPBUGS-2789[*OCPBUGS-2789*])  
```

Comment:
I would say that the attribute is not supported instead of the load balancers.
```suggestion
* `loadBalancerSourceRanges` attribute is not supported, and thus ignored, in load balancer type services in clusters that run on {rh-openstack} and use the OVN Octavia provider. There is no workaround for this issue. (link:https://issues.redhat.com/browse/OCPBUGS-2789[*OCPBUGS-2789*])  
```

Reply:
I'm also wondering if the linked jira issue should point to an issue in OCP and not to a docs issue which will be solved.

Reply:
@eurijon Are you aware of this being tracked elsewhere? I've got nothing.

Reply:
```suggestion
* The `loadBalancerSourceRanges` attribute is not supported, and is thus ignored, in load-balancer type services in clusters that run on {rh-openstack} and use the OVN Octavia provider. There is no workaround for this issue. (link:https://issues.redhat.com/browse/OCPBUGS-2789[*OCPBUGS-2789*]) 
```

Reply:
@luis5tb's explanation in OCPBUGS-2789 now makes this link more useful.

Reply:
It looks good as loadBalancerSourceRanges is not supported with OVN-octavia and an epic will be created for addressing it as stated in https://issues.redhat.com/browse/OCPBUGS-2789